go.mod

```diff
@@ -1,15 +1,15 @@
 module github.com/prometheus-community/postgres_exporter
 
-go 1.14
+go 1.16
```

Comment:
We don't usually change this unless we need to for language feature reasons. Does this not build on 1.14 or 1.15 anymore?

Reply:
I think I changed this trying to figure out why illumos wouldn't build. I'll roll it back. Thanks for the catch.